packages/app-store/zohocalendar/_metadata.ts

```diff
@@ -0,0 +1,21 @@
+import type { AppMeta } from "@calcom/types/App";
```

Comment:
Replacing this file with config.json. It is deprecated and AppStore CLI doesn't create it.